ModConfigMenu/ModConfigMenu/Src/ModConfigMenuAPI/Classes/MCM_API_Instance.uc

```diff
@@ -1,5 +1,13 @@
 interface MCM_API_Instance;
 
+enum eSettingEventMode
+{
+    // Callbacks are triggered when the user manipulates the UI element. Use SetSaveHandler and SetCancelHandler to save/abort.
+    eSettingEventMode_Immediate,

```

Comment:
Are you suggesting that each control have a separate OnChanged and OnSave callback?